Library/Client.ts

```diff
@@ -277,8 +278,12 @@ class Client {
 
         if (telemetryProcessorsCount === 0) {
             return accepted;
-        }
+        }     
 
+        var correlationContext = CorrelationContextManager.getCurrentContext();
+        var userCorrelationContext = correlationContext && correlationContext.userProperties;
+        contextObjects = contextObjects || {};
+        contextObjects['userProperties'] = userCorrelationContext;
```

Comment:
Correlation context has two fields: 'operationId' and 'userProperties'. `userCorrelationContext` can be either a whole `correlationContext` object or just `userProperties` (like 284). If you assign it to `userProperties' here, wouldn't it duplicate the `operationId'?